crap-advisor-frontend/src/app/typings/index.ts

```diff
@@ -0,0 +1,28 @@
+export type TLocation = {
+  latitude: number;
+  longitude: number;
+};
+
+export interface Restroom {
+  id: number;
+  created: Date;
+  location: TLocation;
+  name: string;
+  rating: number | null;
+}
+
+export type ReviewMarks = {
+  cleanness: number;
```

Comment:
All marks can be null, should we specify that like at line 11?

Reply:
Yeah, you right. I have no documetation and that's a reason why i type this variable like that

Reply:
They are nullable in swagger, btw i found some mistakes in other parts of the documentation, i'll fix it

Reply:
Cool. Please notify me after fix.